packages/shared/src/models/objection/FileMeta.ts

```diff
@@ -27,7 +29,8 @@ export default class File extends BaseModel {
     if (this.owner) this.owner = this.owner.toLowerCase() // sanitize address
     // expire in 1 day without verification
     if (!this.expiresAt) this.expiresAt = new Date(Date.now() + EXPIRATION_PERIOD)
-    // to-do: set lastScannedBlock as current block
+    // set lastScannedBlock as current block
+    if (!this.lastScannedBlock) this.lastScannedBlock = await etherscan.getBlockNumber()
```

Comment:
What do you think about adding a safety window here, maybe scanning the past 5/10 blocks makes sense

Reply:
Yeah makes sense, added `LAST_SCANNED_BLOCK_OFFSET = 10` constant